### DISCUSSION TITLE
Running Bruno in ADO

### DISCUSSION THREAD

**Discussion by ggriskow** [usebruno/bruno#1902] [Q&A]
Hi All,

New to Bruno coming from Postman. So far able to do everything I need locally on my machine. Next step is to try to run the collection in an ADO pipeline. I'm seeing 2 potential issues, 1 - Bruno CLI needs to run from the folder of the collection, 2 - Bruno stores all the files for a collection in a folder unlinke Postman. Any help would be greatly appreciated.

Thanks
Greg

**Comment by ggriskow** [comment]
I have worked my way through this. I am able to cd to the folder to run Bruno in ADO and just need to copy the complete folders including the environements folder into ADO as well.

**Reply by ashameem123** [reply]
Could you please elaborate how to checkin code in ADO ?

**Comment by sanjai0py** [comment]
Hey @ggriskow, glad to hear that you found a solution! Could you please share a how-to guide for others who may be facing the same issue? Thank you!
